agentops/langchain_callback_handler.py

```diff
@@ -38,10 +39,12 @@ def __init__(self, api_key: str,
         self.ao_client = AOClient(**{k: v for k, v in client_params.items()
```

Comment:
i know i wrote this initially but we should just use `*args, **kwargs` here. otherwise every time the client constructor args change, this will have to too

Reply:
vscode doesnt pass args through